config/initializers/sentry.rb

```diff
@@ -7,5 +7,31 @@
 
     # Send 5% of transactions for performance monitoring
     config.traces_sample_rate = 0.05
-  end
+    config.enabled_patches += [:sidekiq_cron]
```

Comment:
We are using `sidekiq-scheduler` so this needs to be;
```suggestion
    config.enabled_patches += [:sidekiq_scheduler]
```